Example Apps/Example Swift/RemoteControlManager.swift

```diff
@@ -0,0 +1,237 @@
+//
+//  RemoteControlManager.swift
+//  SmartDeviceLink-Example-Swift
+//
+//  Created by Beharry, Justin (J.S.) on 7/28/22.
+//  Copyright © 2022 smartdevicelink. All rights reserved.
+//
+
+import Foundation
+import SmartDeviceLink
+import SmartDeviceLinkSwift
+
+class RemoteControlManager {
+    private let sdlManager: SDLManager
+    private let homeButtons: [SDLSoftButtonObject]
+    private var remoteControlCapabilities: SDLRemoteControlCapabilities?
+    private var climateModuleId: String?
+    private var hasConsent: Bool?
+    private var climateData: SDLClimateControlData?
+
+    public let isPermissionEnabled: Bool
+    public var climateDataString: String {
+            """
+            AC: \(optionalNumberBoolToString(climateData?.acEnable))
+            AC Max: \(optionalNumberBoolToString(climateData?.acMaxEnable))
+            Auto Mode: \(optionalNumberBoolToString(climateData?.autoModeEnable))
+            Circulate Air: \(optionalNumberBoolToString(climateData?.circulateAirEnable))
+            Climate: \(optionalNumberBoolToString(climateData?.climateEnable))
+            Current Temperature: \(optionalTemperatureToString(climateData?.currentTemperature))
+            Defrost Zone: \(optionalSDLEnumToString(climateData?.defrostZone?.rawValue))
+            Desired Temperature: \(optionalTemperatureToString(climateData?.desiredTemperature))
+            Dual Mode: \(optionalNumberBoolToString(climateData?.dualModeEnable))
+            Fan Speed: \(optionalNumberToString(climateData?.fanSpeed))
+            Heated Mirrors: \(optionalNumberBoolToString(climateData?.heatedMirrorsEnable))
+            Heated Rears Window: \(optionalNumberBoolToString(climateData?.heatedRearWindowEnable))
+            Heated Steering: \(optionalNumberBoolToString(climateData?.heatedSteeringWheelEnable))
+            Heated Windshield: \(optionalNumberBoolToString(climateData?.heatedWindshieldEnable))
+            Ventilation: \(optionalSDLEnumToString(climateData?.ventilationMode?.rawValue))
+            """
+    }
+
+    /// Initialize the RemoteControlManager Object
+    ///
+    /// - Parameters:
+    ///     - sdlManager: The SDL Manager.
+    ///     - permission: Permission from the proxy manager to access remote control data
+    ///     - homeButton: An array of SDLSoftButtonObjects that remote control manager can reset to.
+    init(sdlManager: SDLManager, permission: Bool, homeButtons: [SDLSoftButtonObject]) {
+        self.sdlManager = sdlManager
+        self.isPermissionEnabled = permission
+        self.homeButtons = homeButtons
+    }
+
+    func start() {
+        if (!self.isPermissionEnabled) {
+            SDLLog.d("Missing permissions for Remote Control Manager. Example remote control works only on TCP.")
+            return
```

Comment:
If you'd like, you could clean up the file a little, but returning void functions on one line like this. `return SDLLog.d("A log")` (same goes for the `AlertManager` call). I'll leave it as optional though.